ParabolInc/parabol
# feat: Release team insights

Fixes: #9301 
Reverse the feature flag.

## Testing scenarios

Team insights are generated at the end of a meeting.
- run at least 1 additional retro
- see team insights appear

## Final checklist
